packages/react-jsx-highcharts/src/components/Series/Series.js

```diff
@@ -47,10 +43,7 @@ class Series extends Component {
 
     let needsRedraw = false;
     // Using setData is more performant than update
-    if (isImmutable(data) && immutableEqual(data, prevProps.data) === false) {
-      this.series.setData(data.toJS(), false);
```

Comment:
I feel uneasy about dropping support for Immutable, I can't see how these few lines effect the bundle size that much, and Immutable is quite likely to be used in a React app.

Reply:
It's not about bundle size, it's only a couple of kB. I just thought that making equality checks is unnecessary. 

I can bring it back with immutable 4?

What about isEqual checks? I find it more inline with other libraries when doing just shallow compare?

Reply:
Actually if we keep immutable and isEqual, there is no need to make major release if Highcharts 7 does not break things.

All the new features and lodash changes can be made within version 3.

Might make more sense actually, as most React libraries seem to target their breaking changes to React 16.7.

I am not against that all. What do you think?